perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_group_attribute_def_virt_autoRegistrationEnabled.java

```diff
@@ -0,0 +1,22 @@
+package cz.metacentrum.perun.core.impl.modules.attributes;
+
+import cz.metacentrum.perun.core.api.Attribute;
+import cz.metacentrum.perun.core.api.AttributeDefinition;
+import cz.metacentrum.perun.core.api.Group;
+import cz.metacentrum.perun.core.api.exceptions.WrongAttributeAssignmentException;
+import cz.metacentrum.perun.core.api.exceptions.WrongAttributeValueException;
+import cz.metacentrum.perun.core.api.exceptions.WrongReferenceAttributeValueException;
+import cz.metacentrum.perun.core.impl.PerunSessionImpl;
+import cz.metacentrum.perun.core.implApi.modules.attributes.GroupVirtualAttributesModuleAbstract;
+import cz.metacentrum.perun.core.implApi.modules.attributes.GroupVirtualAttributesModuleImplApi;
+
+public class urn_perun_group_attribute_def_virt_autoRegistrationEnabled extends GroupVirtualAttributesModuleAbstract implements GroupVirtualAttributesModuleImplApi {
```

Comment:
I forgot to mention this. Usually, we also override the `getAttributeDefinition` method. It should return the same definition, which you have used in the `initialize` method.

Also, please write a simple unit test for this module.